requirements_dev.txt

```diff
@@ -6,3 +6,5 @@ pytest>=5.4.1
 pytest-benchmark
 pytest-benchmark[histogram]
 setuptools>=41.0.0
+pygal
+pygaljs
```

Comment:
where is this required?

Reply:
pytest-benchmark seems to require them, I use requirements_dev.txt
 for bazel depends too.